CHANGELOG.md

```diff
@@ -1,3 +1,91 @@
+# v1.11.0 (2022-11-15)
+
+## OS Changes
+
+* Prevent a panic `early-boot-config` when there is no IMDS Region ([#2493])
+* Update grub to 2.06-42 ([#2503])
+* Allow interface configuration via Ethernet permanent address ([#2519])
```

Comment:
This will likely be read as `net.toml` supporting this feature, which it doesn't - it's just a patch we added to wicked to prepare for that functionality.

```suggestion
* Bring back wicked support for matching interfaces via hardware address ([#2519])
```